fuzz/fuzz_targets/markdown.rs

```diff
@@ -9,7 +9,15 @@ fuzz_target!(|data: &[u8]| {
             s,
             &markdown::Options::gfm()
         );
+        let _ = markdown::to_html_with_options(
+            s,
+            &markdown::Options {
+                parse: markdown::ParseOptions::mdx(),
+                ..Default::default()
+            }
+        ).ok();
```

Comment:
This is pretty useless: MDX when going to HTML.
The latter one is useful: parsing MDX to an AST

Reply:
removed